aprslib/parsing/thirdparty.py

```diff
@@ -0,0 +1,21 @@
+import re
+from aprslib.parsing.__init__ import parse
+from aprslib.exceptions import UnknownFormat
+from aprslib.exceptions import ParseError
+
+__all__ = [
+        'parse_thirdparty',
+        ]
+
+def parse_thirdparty(body):
+    parsed = {'format':'Third party'}
```

Comment:
format name doesn't follow the standard set by the package. That is lower case and no spaces.